sydent/sydent.py

```diff
@@ -78,6 +79,7 @@ def list_from_comma_sep_string(rawstr):
         'shadow.hs.master': '',
         'shadow.hs.slave': '',
         'ips.nonshadow': '',  # comma separated list of CIDR ranges which /info will return non-shadow HS to.
+        'invites.validity_period': 0,
```

Comment:
again, None seems like a more sensible sentinel value

Reply:
please can it have more comments, including what it means, and what the format is?